packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -21,6 +21,24 @@ import {
 import S3 from "aws-sdk/clients/s3";
 import { performance } from "perf_hooks";
 
+const parseCookieValue = (cookie: string) => {
```

Comment:
I was hoping we could use express cookie-parser (https://github.com/jshttp/cookie) here. It's benchmarked and battle tested. Even if it means to [borrow the code parser](https://github.com/jshttp/cookie/blob/243fa7edab7f213a6af8adb339e9530daa23b445/index.js#L49).